eng/scripts/Language-Settings.ps1

```diff
@@ -6,11 +6,17 @@ $packagePattern = "*.nupkg"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/dotnet-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=dotnet%2F&delimiter=%2F"
 
-function Get-dotnet-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
+function Get-dotnet-PackageInfoFromRepo ($pkgDirectoryPath, $serviceDirectoryName)
 {
-  $projectPath = Join-Path $pkgPath "src" "$pkgName.csproj"
-  if (Test-Path $projectPath)
+  $pkgProperties = @()
+  if (Test-Path (Join-Path $pkgDirectoryPath "src"))
   {
+    $projectFilePaths = Get-ChildItem (Join-Path $pkgDirectoryPath "src") "*.csproj" -File
```

Comment:
If we ever have a case where there is more then one project I think we are in an error case.

Reply:
Why is that?
Wouldn't it just be filtered out with the ArtifactList in ci.yml